python/docs/linked_list_implementation/README.md

```diff
@@ -1,11 +1,13 @@
 # Singly Linked List
-<!-- Short summary or background information -->
+A singly linked list is a sequence of Nodes which are connected to each other. Singly linked lists means there is only
+one reference for each node unlike its brother doubly linked lists which has a reference to both the next and previous
 
 ## Challenge
-
+This challenge asks us to build to classes. One is a linked list class which is a signly linked list connecting a series
+nodes by just one reference, "next". The other class we create is a Node class which is each individual index of a
+linked list
 
 ## Approach & Efficiency
 <!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
-
-## API
```

Comment:
For the API section, talk about what the methods are and how to use them. For example:

- `insert()` -> `None`. Adds a new node with that value to the head of the list with an O(1) Time performance.
- ` includes(value)` -> `Boolean`. Indicates whether that value exists as a Node’s value somewhere within the list.
- `to_string()` -> `String`. A string representing all the values in the Linked List, formatted as: "{ a } -> { b } -> { c } -> NULL"